### ISSUE TITLE
Visibility issue

### ISSUE THREAD

**Issue by wwwpAT** [beto-rodriguez/Live-Charts#298]
Hi!   
(Edit: --- I'm using LiveCharts 0.7.11)

First of all, thank you for your awesome charts. I love them!

At the moment I have a SeriesColleciton bound to a chart containing ALL my series (200+):
Each LineSeries.Visibility is connected to a checkbox, so I can display/hide each one of them.

I initialize each LineSerie with 
`Visibility = System.Windows.Visibility.Collapsed`

Reproducing my issue:
1.: Start the application - Chart is empty (fine)
**2.: Select Checkbox - Chart remains empty (I expect the series to show up)**
3.: Change size of chart -> triggers some update - Chart displays all selected Series
4.: Unselect Checkbox - Series disappear (fine)
5.: Select Checkbox - Series appear (fine)

Whenever I select a series which hasn't been displayed before, I need to resize the chart. 
This triggers the chart to update and all selected series show up.
Then everything works just fine.
Did I do something wrong?

I made a small sample which can show the behavior.
All series are un-/selected by a single ComboBox (to keep the example small).

XAML: LiveChart with one ComboBox
```
<Window x:Class="Playground.MainWindow"
        x:Name="AppRoot" DataContext="{Binding ElementName=AppRoot}"
        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:local="clr-namespace:Playground"      
        xmlns:lvc="clr-namespace:LiveCharts.Wpf;assembly=LiveCharts.Wpf"
        mc:Ignorable="d"
        Title="MainWindow" Height="350" Width="525">
    <Grid>
        <Grid.RowDefinitions>
            <RowDefinition Height="auto"/>
            <RowDefinition Height="*"/>
        </Grid.RowDefinitions>
        <CheckBox Grid.Row="0" Checked="CheckBox_Checked" Unchecked="CheckBox_Unchecked" Content="Display all series"/>
        <lvc:CartesianChart Grid.Row="1"
                     Background="#222E31" Series="{Binding Model.MySeriesCollection}" DisableAnimations="True">
            <lvc:CartesianChart.AxisY>
                <lvc:Axis>
                    <lvc:Axis.Separator>
                        <lvc:Separator Step="1"></lvc:Separator>
                    </lvc:Axis.Separator>
                </lvc:Axis>
            </lvc:CartesianChart.AxisY>
            <lvc:CartesianChart.AxisX>
                <lvc:Axis MinValue="0"></lvc:Axis>
            </lvc:CartesianChart.AxisX>
        </lvc:CartesianChart>
    </Grid>
</Window>
```

XAML.cs
```
using LiveCharts.Wpf;
using System.Windows;

namespace Playground
{
    public partial class MainWindow : Window
    {
        public MainWindow()
        {
            InitializeComponent();
            Model = new ViewModel();
        }

        public static readonly DependencyProperty ModelProperty = DependencyProperty.Register("Model", typeof(ViewModel), typeof(MainWindow),
           new PropertyMetadata(default(ViewModel)));
        public ViewModel Model { get { return (ViewModel)GetValue(ModelProperty); } set { SetValue(ModelProperty, value); } }

        /// <summary>
        /// Display all Series
        /// </summary>
        /// <param name="sender"></param>
        /// <param name="e"></param>
        private void CheckBox_Checked(object sender, RoutedEventArgs e)
        {
            foreach(LineSeries mySerie in Model.MySeriesCollection)
            {
                mySerie.Visibility = Visibility.Visible;
            }
        }

        /// <summary>
        /// Hide all series
        /// </summary>
        /// <param name="sender"></param>
        /// <param name="e"></param>
        private void CheckBox_Unchecked(object sender, RoutedEventArgs e)
        {
            foreach (LineSeries mySerie in Model.MySeriesCollection)
            {
                mySerie.Visibility = Visibility.Collapsed;
            }
        }
    }
}
```

My ViewModel 
using NuGet Package "PropertyChanged.Fody"
```
using System;
using System.ComponentModel;
using System.Runtime.CompilerServices;
using PropertyChanged;
using LiveCharts;
using LiveCharts.Wpf;

namespace Playground
{
    [ImplementPropertyChanged]
    public class ViewModel : INotifyPropertyChanged
    {
        public event PropertyChangedEventHandler PropertyChanged;

        protected virtual void OnPropertyChanged([CallerMemberName] string propertyName = null)
        {
            if (PropertyChanged != null)
            {
                PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
            }
        }

        public ViewModel()
        {
            Random R = new Random();
            for (int i = 0; i < 5; i++)
            {
                MySeriesCollection.Add(new LineSeries()
                {
                    Values = new ChartValues<int> { R.Next(10), R.Next(10), R.Next(10), R.Next(10), R.Next(10), R.Next(10) },
                    Fill = System.Windows.Media.Brushes.Transparent,
                    Title = "Series " + i,
                    LineSmoothness = 0,
                    Visibility = System.Windows.Visibility.Collapsed
                    
                });
            }
        }
        public SeriesCollection MySeriesCollection { get; } = new SeriesCollection();
    }
}
```

**Comment by NickSavin** [comment]
Hi, Beto!
I Have got the similar issue. The Visibility changes has no effect, if lineSeries is initialized with Visibility.Hidden

**Comment by NickSavin** [comment]
its an animation with example 

![123](https://cloud.githubusercontent.com/assets/20172765/17887211/212ec372-692e-11e6-886a-de7e9e43588a.gif)

**Comment by beto-rodriguez** [comment]
@wwwpAT you were doing nothing wrong, sorry for the late answer.

this was wrong. it last commit fixes it

**Comment by beto-rodriguez** [comment]
I will include this fix in 0.7.12 version, you can also close and let me know if this does not solves your issue, thanks for reporting
